packages/coordinator/ts/sessionKeys/sessionKeys.service.ts

```diff
@@ -64,26 +62,22 @@ export class SessionKeysService {
    * Generate a KernelClient from a session key and an approval
    *
    * @param sessionKeyAddress - the address of the session key
-   * @param encryptedApproval - the encrypted approval string
+   * @param approval - the approval string
    * @param chain - the chain to use
-   * @returns
+   * @returns a KernelAccountClient
    */
   async generateClientFromSessionKey(
     sessionKeyAddress: Hex,
-    encryptedApproval: string,
-    chain: Chain,
+    approval: string,
+    chain: ESupportedNetworks,
```

Comment:
why we don't need to encrypt the approval now?

Reply:
so technically the approval is just a string, without the session key private key one cannot do anything. So even if the approval is stolen they cannot do anything, and the temporary private key never leaves the server, so if they had access to the private key, they would also be able to get the approval whether it's encrypted or not